torch/distributed/_tensor/ops/math_ops.py

```diff
@@ -36,6 +36,9 @@ def default_reduction_rule(op_schema: OpSchema) -> OutputSharding:
     [
         aten.sum.default,
         aten.sum.dim_IntList,
+        aten.mean.default,
+        aten.mean.dim,
+        aten.mean.out
     ]
 )
 def sum_rule(op_schema: OpSchema) -> OutputSharding:
```

Comment:
since this rule is not only for `sum`, we can rename it to something like `linear_reduction_rule`.

Reply:
sure, will do it later